book/05-distributed-git/sections/contributing.asc

```diff
@@ -535,7 +535,7 @@ $ git remote add myfork <url>
 
 Anschließend müssen Sie Ihre neue Arbeit in dieses Repository hochladen.
 Es ist am einfachsten, den Branch, an dem Sie arbeiten, in Ihr geforktes Repository hochzuladen, anstatt diese Arbeit in Ihrem ´master´-Branch zusammenzuführen und diesen hochzuladen.
-Der Grund dafür ist, dass Sie Ihren `master`-Branch nicht zurücksetzen müssen, wenn Ihre Arbeit nicht akzeptiert bzw. nur teilweise akzeptiert (cherry-pick) wurde (die Git-Operation zum `cherry-pick` wird ausführlicher in <<ch05-distributed-git#_rebase_cherry_pick>> behandelt).
+Der Grund dafür ist, dass Sie Ihren `master`-Branch nicht zurücksetzen müssen, wenn Ihre Arbeit nicht akzeptiert bzw. nur übernommen akzeptiert (cherry-pick) wurde (die Git-Operation zum `cherry-pick` wird ausführlicher in <<ch05-distributed-git#_rebase_cherry_pick>> behandelt).
```

Comment:
Verbessere mich bitte, wenn ich falsch liege:
Ich habe "cherry-picking" so verstanden, dass Teile des Pull-Requests akzeptiert und andere abgelehnt werden.
Wenn ich da richtig liege, müsste im Satz etwas wie "teilweise" auftauchen.
```suggestion
Der Grund dafür ist, dass Sie Ihren `master`-Branch nicht zurücksetzen müssen, wenn Ihre Arbeit nicht akzeptiert wurde bzw. nur einzelne Commits übernommen (cherry-pick) wurden (die Git-Operation zum `cherry-pick` wird ausführlicher in <<ch05-distributed-git#_rebase_cherry_pick>> behandelt).
```

Reply:
In 87bddea5506c6c5a406307cc4f58bae5959c33a4 wurde hierfür bereits was angepasst.

Reply:
Ja, hatte ich nachträglich auch bemerkt (s. meinen EDIT-Kommentar) im vorherigen Beitrag

Reply:
Ah, OK dann hatte ich dich nicht richtig verstanden. Dam markiere ich dies hier mal als Resolved.